### DISCUSSION TITLE
Cannot Sign Up For Vercel Account

### DISCUSSION THREAD

**Discussion by barziahmed** [vercel/community#6153] [Account Issue]
### Question

Hello, 

I just can't complete signing up for Vercel, it's extremely fruastrating. 

When I write my email for singing up and verify that email, it asks for my phone number, then when I write my phone number and click continue, I get the following error message.

![t](https://github.com/vercel/community/assets/135956133/60fb7f9b-f1f7-4bbb-890a-865077fff0c7)

**Comment by amyegan** [comment]
Hey @Barzi-Ahmed. Please email [registration@vercel.com](mailto:registration@vercel.com) to open a case if you haven't already. I can't safely verify your account information here in public, but that email will get you in touch with our registration experts 🙏
